source/fundamentals/authentication.txt

```diff
@@ -93,7 +93,7 @@ see the corresponding syntax for specifying the default authentication mechanism
       .. code-block:: csharp
 
          var credentials = MongoCredential.CreateCredential("<authenticationDb>", "<username>", "<password>");
-         var settings = MongoClientSettings.FromConnectionString("<host>");
+         var settings = MongoClientSettings.FromConnectionString("<connection string>");
          settings.Credential = credentials;
```

Comment:
Suggestion: Change to `credential` on lines 95 and 97 to match every other example.